.github/workflows/FormatCheck.yml

```
@@ -23,7 +23,7 @@ jobs:
       - uses: julia-actions/cache@v2
       - name: Install JuliaFormatter and format
         run: |
-          julia  -e 'using Pkg; Pkg.add(PackageSpec(name = "JuliaFormatter", version="1.0.45"))'
+          julia  -e 'using Pkg; Pkg.add(PackageSpec(name = "JuliaFormatter", version="1.0.6"))'
```

```suggestion
          julia  -e 'using Pkg; Pkg.add(PackageSpec(name = "JuliaFormatter", version="1.0.60"))'
```
I assume you wanted to use v1.0.60, didn't you?

---

Actually yes, but then I get an error message. I downgraded the utility script. I have no time now to debug that as well.

---

Phew ... I get the same error. So it is not the version ... merde...

---

Looks fine now, doesn't it?

---

Yeahh ... the problem was in front of the computer ...